examples/kprobe/fn_amd64.go

```diff
@@ -0,0 +1,4 @@
+package main
+
+// Name of the kernel function to trace.
+const fn = "__x64_sys_execve"
```

Comment:
Checking https://golang.org/pkg/runtime/#pkg-constants GOARCH in the code should be simpler.

Reply:
OK, changed the approach. Let me know if that works.